pkg/agent/qrm-plugins/cpu/dynamicpolicy/state/util.go

```diff
@@ -146,10 +149,16 @@ func GetSharedQuantityMapFromPodEntries(podEntries PodEntries, ignoreAllocationI
 			}
 
 			if poolName := allocationInfo.GetOwnerPoolName(); poolName != advisorapi.EmptyOwnerPoolName {
-				ret[poolName] += GetContainerRequestedCores()(allocationInfo)
+				preciseQuantityMap[poolName] += GetContainerRequestedCores()(allocationInfo)
 			}
 		}
 	}
+
+	for poolName, preciseQuantity := range preciseQuantityMap {
```

Comment:
will it exceeds node total if all pools are ceiled?

Reply:
yes, the sum of ret will exceed node total here. but the generatePoolsAndIsolation will adjust to guarantee cpuset of every pool won't exceed current available cpus.